travispy/entities/hook.py

```diff
@@ -30,3 +31,29 @@ class Hook(Entity):
         'private',
         'admin',
     ]
+
+    def _set_hook(self, flag):
```

Comment:
As I mentioned in the original issue, the idea is adding these hook methods inside ``Repo`` entity not here at ``Hook``, mainly because the Ruby solution already does like this.